flixel/math/FlxMath.hx

```diff
@@ -573,4 +573,13 @@ class FlxMath
 	{
 		return (n > 0) ? n : -n;
 	}
+	
+	/**
+	 * Returns `a mod b`.
```

Comment:
The doc should explain what "mod" is, the difference between this and haxe's `%` operator, and give at least one key example

Reply:
also we should add that example and others to the [unit test](https://github.com/HaxeFlixel/flixel/blob/dev/tests/unit/src/flixel/math/FlxMathTest.hx)